lib/rss.js

```diff
@@ -25,8 +25,11 @@ module.exports = ({
   feed_items: current,
   data_updated: dataUpdated,
 }) => {
-  const oldOrgs = createOrgHash(oldData)
   const newOrgs = createOrgHash(newData)
+  let oldOrgs = newOrgs
+  if (oldData) {
+    oldOrgs = createOrgHash(oldData)
+  }
```

Comment:
Actually, we could do this:

```javascript
const oldOrgs = createOrgHash(oldData) || createOrgHash(newdata)
```

Please not that the return should be falsy object such as false boolean, undefined, null, or zero.

Reply:
We can't do that because if we pass an undefined oldData to createOrgHash, it would throw an exception. We could wrap the function inside a try/catch block, but I'm not sure if that's better than using an if statement.

Reply:
You may use `createOrgHash(oldData || 'something')`. As I said, it's trivial.

Reply:
We could use ternary if we really want it short

```js
const oldOrgs = oldData ? createOrgHash(oldData) : newOrgs
```

but, maybe, it's clearer if we use explicit conditional.